### PR TITLE
Update pypy in travis

### PR DESCRIPTION
**Pull request by pganssle**
## Summary of changes

Attempting to update the Travis configuration to use a newer version of `pypy2.7` by updating to xenial, per @webknjaz's suggestion. Not sure this will actually work, so I've marked it WIP.

Trying to debug or solve #1730

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,7 +8,8 @@ jobs:
     python: 2.7
   - <<: *latest_py2
     env: LANG=C
-  - python: pypy
+  - python: pypy2.7
```
Comment (webknjaz) [inline .travis.yml]: You should set a specific version, otherwise you might actually get something older.
```suggestion
  - python: pypy2.7-6.0.0
```

**Comment by pganssle** [comment]
Huzzah, that "fixed" it!

**Comment by webknjaz** [comment]
btw here's a place to complain about 7.0 and 7.1: https://travis-ci.community/t/add-pypy-7-support/2228

**Comment by pganssle** [comment]
I've updated the commit message and the CI is fixed, but I will leave #1730 open a bit longer in case we can figure out the actual root cause.

Thanks for the quick fix @webknjaz!

**Pull request merged**